### PR TITLE
Fix/results type error

### PR DESCRIPTION
**Pull request by mdip226**
:)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -7,6 +7,9 @@ dill
 requests
 dask_kubernetes
 jupyter
+blosc
+lz4
+pandas
```
Comment (ethangreen-dev) [inline requirements.txt:12]: What's the need for these additional dependencies?

File: stochss_compute/api/v1/memory.py [hunk 2]
```diff
@@ -66,7 +67,7 @@ def make_plot(memory_id: str):
         return "Something broke", 404
 
     # Grab the memory.
-    memory: Results = delegate.job_results(memory_id)
+    memory: Results = Jsonify.from_json(delegate.job_results(memory_id))
```
Comment (ethangreen-dev) [inline stochss_compute/api/v1/memory.py:70]: I'm not sure if 
`Jsonify#from_json` will deserialize the JSON to the right type. Better to be explicit and use `Results#from_json`.

**Review by ethangreen-dev — CHANGES_REQUESTED** [review]
Needs a couple changes / clarifications. Otherwise it looks good! :)

File: stochss_compute/api/v1/memory.py (hunk 2 above)
Comment (mdip226) [reply stochss_compute/api/v1/memory.py:70]: ah, yeah

File: requirements.txt (hunk 1 above)
Comment (mdip226) [reply requirements.txt:12]: ok, I am now relatively sure that the only one of these that we need is blosc. For the error that it fixes, refer to this
[logs-from-stochss-compute-in-stochss-compute-6cff5f86df-6p4qf.txt](https://github.com/StochSS/stochss-compute/files/7487581/logs-from-stochss-compute-in-stochss-compute-6cff5f86df-6p4qf.txt)

Comment (mdip226) [reply requirements.txt:12]: But to answer your question, I don't know. It seems weird to me that this package was not already included in one of the other dask packages

**Review by ethangreen-dev — APPROVED** [review]

**Pull request merged**